pkg/node/sendoperation/sendoperation.go

```diff
@@ -207,3 +208,16 @@ func DecodeMessage64(msgB64 string) ([]byte, uint64, uint64, error) {
 
 	return decodedMsg, fee, expiry, nil
 }
+
+// DecodeOperationID decodes a byte slice to retrieve the operation ID.
+func DecodeOperationID(data []byte) (uint64, error) {
```

Comment:
I think we should have it. By instance You can copy the opId and search it on the explorer

Reply:
Actually the opID HERE IS just an identifier for the type of operation : 0 for transfer , 1 for buy roll , 2 for sell roll , .... , it will help us in the wallet to identify the type of operation before starting the decoding, as for now , we suppose blindly that all operations are call sc , and if an error occurred during decoding, we make the next assumption : execute , ...